config/settings/base.py

```diff
@@ -134,11 +134,6 @@
 ]
 # https://docs.djangoproject.com/en/dev/ref/settings/#auth-password-validators
 AUTH_PASSWORD_VALIDATORS = [
-    {
-        "NAME": "django.contrib.auth.password_validation."
-        "UserAttributeSimilarityValidator",
```

Comment:
Would it be possible to subclass the [`UserAttributeSimilarityValidator`](https://github.com/django/django/blob/8c279113864c31cef539c6384a683e6ed832adca/django/contrib/auth/password_validation.py#L183-L213) to ensure it checks against the user's name to our satisfaction? Based on how it's used here, it seems as though the `max_similarity` threshold could be lowered to match Amanda's expectations.

Reply:
We were including the name, email and username fields, with `max_similarity` set to `0.7`, and we can configure those options in settings without subclassing. I guess we'd want to turn the similarity down.

Reply:
I turned it down to `0.6`. At `0.5`, it was saying `squarelet` was too similar to `doe@example.com`.